crates/api_common/src/build_response.rs

```diff
@@ -141,6 +140,16 @@ pub async fn send_local_notifs(
     }
   };
 
+  let inbox_link = format!("{}/inbox", context.settings().get_protocol_and_hostname());
+  let comment_link = |comment: &Comment| {
+    format!(
+      "{}/post/{}/{}",
```

Comment:
Would probably be better to use the `/comment/id` rather than the one that includes the post. 

I'd also prefer this be a function on the Comment impl in `crates/db_schema/src/impls/comment.rs` . Just pass in the protocol and hostname as a &str . 

IE `pub fn link(&self, protocol_and_hostname: &str) -> ...`

Reply:
Made a separate PR: https://github.com/LemmyNet/lemmy/pull/5394